src/WebJobs.Script.WebHost/Standby/StandbyManager.cs

```diff
@@ -209,6 +209,19 @@ private async Task CreateStandbyWarmupFunctions()
             File.WriteAllText(Path.Combine(functionPath, "run.csx"), content);
 
             _logger.LogInformation("StandbyMode placeholder function directory created");
+
+            ReadRuntimeAssemblyFiles();
+        }
+
+        private void ReadRuntimeAssemblyFiles()
```

Comment:
I think it's better to put this method in HostWarmupMiddleware and run it next to PGO, ... if it is a warmup request. that way if we add more logic for warmup it can all be in same location